app/views/pages/home.html.erb

```diff
@@ -102,9 +89,8 @@
         <h5 class="center">Nurturing Space</h5>
 
         <p class="light">
-          Our active community of military and veterans open source software &amp; hardware enthusiasts help you gain
-          real-world app experience, share job opportunities, conference scholarships, and opportunities to build on your
-          military experience.
+          Our growing community of software enthusiasts will help you learn about code schools, scholarships,
+          jobs, gain real-world programming experience, and more.
```

Comment:
i think "career opportunities" is better than jobs. I don't think the 'by contributing to open source' is a plus because only some will know what that is. The goal of homepage copy is to stick to the benefits folks receive, as simply as possible without confusing anybody, and while writing as little as possible.